### DISCUSSION TITLE
Allow rules to define what URL opens in what workspace

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#786] [Ideas]
### What feature would you like to see?

The workspaces feature would be much more complete if it is possible to add url filters that link to certain workspaces. For instance:
- if url contains azure.com -> use work
- if url contains youtube.com -> use private

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/600_

**Comment by prometheas** [comment]
Would absolutely love to see this one get some attention 🙏
